### ISSUE TITLE
Graph reset when using conditional color

### ISSUE THREAD

**Issue by pawamoy** [brndnmtthws/conky#441]
Hi, has this issue been fixed: https://sourceforge.net/p/conky/bugs/379/?
I still encounter it with conky 1.10.6.

Observations:

- affects all graphs (mem, diskio, up, down)
- but only the ones that are AFTER a colored graph
- graphs' *alternate* history seems to be kept in subsequent calls (look at the picture, CPU1 has been working more than others)

GIF:
![conky_graph_reset_cond_color](https://user-images.githubusercontent.com/3999221/34172233-a82b332a-e4f1-11e7-9bc6-7e9c2a1a8534.gif)

Picture:
![conky_subsequent](https://user-images.githubusercontent.com/3999221/34172547-b681f764-e4f2-11e7-891c-9536fa8962f4.png)

Here are the relevant conkyrc lines:

```
# CPU Template
template0 ${font}${alignr}CPU\1: ${cpu cpu\1}%  ${platform coretemp.0/hwmon/hwmon1 temp \1}°C  ${if_match ${cpu cpu\1} >= 90}${color9}${else}${if_match ${cpu cpu\1} >= 50}${color3}${endif}${endif}${cpubar cpu\1 5,80}${color}\n${font}${alignr}${cpugraph cpu\1}

TEXT

${font Aerial:style=Bold:pixelsize=10}SYSTEM ${hr 2}${font}
${template0 1}
${template0 2}
${template0 3}
${template0 4}
${font}${alignr}RAM: ${memperc}% (${mem}) ${membar 5,80}
${font}${alignr}${memgraph}
```

And `conky -V`:

```
conky 1.10.6 compiled Mon Jan  2 09:28:09 UTC 2017 for Linux 3.16.0-4-amd64 x86_64

Compiled in features:

System config file: /etc/conky/conky.conf
Package library path: /usr/lib/conky


 General:
  * math
  * hddtemp
  * portmon
  * IPv6
  * Curl
  * RSS
  * Weather (METAR)
  * Weather (XOAP)
  * wireless
  * support for IBM/Lenovo notebooks
  * nvidia
  * eve-online
  * builtin default configuration
  * old configuration syntax
  * Imlib2
  * apcupsd
  * iostats
  * ncurses
  * Internationalization support
  * PulseAudio

 Lua bindings:
  * Cairo
  * Imlib2
  * RSVG
 X11:
  * Xdamage extension
  * Xinerama extension (virtual display)
  * XDBE (double buffer extension)
  * Xft
  * ARGB visual
  * Own window

 Music detection:
  * Audacious
  * MPD
  * MOC
  * XMMS2

 Default values:
  * Netdevice: eth0
  * Local configfile: $HOME/.conkyrc
  * Localedir: /usr/share/locale
  * Maximum netdevices: 64
  * Maximum text size: 16384
  * Size text buffer: 256
```

**Comment by m-wells** [comment]
I am experiencing this issue as well.

**Comment by lasers** [comment]
Silly bug, indeed.

![cpugraph_reset](https://user-images.githubusercontent.com/852504/43725680-c8b6a004-9962-11e8-8254-4c291449733e.gif)
```lua
conky.config = {
    out_to_x=true,
    own_window=true,
    own_window_colour='272B35',
    out_to_console=false,
    update_interval=0.1
}
conky.text = [[
    ${color 0077ff}${if_match $cpu>90}${color ff0000}$endif${cpubar 5,110}
    ${color 0077ff}${cpugraph 25,180}
]]
```

**Comment by Tomunu** [comment]
This should now be fixed with the changes in #849

**Comment by lasers** [comment]
Closed via #849.

**Comment by Ducvoccer** [comment]
bug is not fixed?
